NFproject/articles/templates/article.html

```diff
@@ -1,10 +1,13 @@
 {% extends 'base.html' %}
 {% block content %}
 <div class="container">
-  <h1> {{article.title}}</h1>
+  <h1> {{article.Title}}</h1>
```

Comment:
Model fields should be lowercase by convention. This should be `article.title`.

Only classes should have a capital letter.